jenkins.sh

```diff
@@ -7,10 +7,11 @@ bundle install --path "${HOME}/bundles/${JOB_NAME}" --deployment
 export RESTCLIENT_LOG="log/smokey-rest-client.log"
 export ENVIRONMENT=${TARGET_PLATFORM}
 
-FLAGS="--profile ${TARGET_PLATFORM}"
+FLAGS=(--profile "${TARGET_PLATFORM}")
+FLAGS+=(-t "not @benchmarking")
```

Comment:
I tried this approach, but it doesn't work because of the complex "not @benchmarking" string, which results in `@benchmarking` being interpreted as an argument, rather than (part of) the value of the `-t` flag.